webapp/src/components/shareBoardComponent.tsx

```diff
@@ -76,7 +76,7 @@ const ShareBoardComponent = React.memo((props: Props): JSX.Element => {
     shareUrl.searchParams.set('r', readToken)
 
     if (match.params.workspaceId) {
-        const newPath = generatePath('/workspace/:workspaceId/shared/:boardId/:viewId', {
+        const newPath = generatePath('boards/workspace/:workspaceId/shared/:boardId/:viewId', {
```

Comment:
This path is also in several other places. Shouldn't we create a const somewhere so when it changes again this won't break?

Reply:
We have part of that in the form of `Utils.getFrontendBaseURL`. I updated this to use that setting instead, which is similar to how it is done [here.](https://github.com/mattermost/focalboard/blob/main/webapp/src/app.tsx#L61)

I don't see this string being used anywhere else. It is similar to those in the above mentioned function, but different due to setting vs. retrieving. I do agree we should probably create constants for all of these in `app.tsx`, so the different variations can get set in the same place but that is beyond the scope of this bug fix.  I will create a card for that.